assets/queries/cloudFormation/aws/default_kms_key_usage/query.rego

```diff
@@ -0,0 +1,55 @@
+package Cx
+
+import data.generic.cloudformation as cldLib
+import data.generic.common as common_lib
+
+CxPolicy[result] { #Resource Type DB  and StorageEncrypted is False
+	document := input.document[i]
+	resource := document.Resources[key]
+    common_lib.inArray({"AWS::DocDB::DBCluster", "AWS::Neptune::DBCluster", "AWS::RDS::DBCluster", "AWS::RDS::DBInstance", "AWS::RDS::GlobalCluster"}, resource.Type)
+
+    properties := resource.Properties
+	cldLib.isCloudFormationFalse(properties.StorageEncrypted)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("Resources.%s.Properties.StorageEncrypted", [key]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("Resources.%s.Properties.StorageEncrypted should be true", [key]),
+		"keyActualValue": sprintf("Resources.%s.Properties.StorageEncrypted is false", [key]),
+	}
+}
+
+CxPolicy[result] { # DBTypes any DB, but without storage encrypted is undefined
+	document := input.document[i]
+	resource := document.Resources[key]
+    common_lib.inArray({"AWS::DocDB::DBCluster", "AWS::Neptune::DBCluster", "AWS::RDS::DBCluster", "AWS::RDS::DBInstance", "AWS::RDS::GlobalCluster"}, resource.Type)
+
+	properties := resource.Properties
+	not common_lib.valid_key(properties, "StorageEncrypted")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("Resources.%s.Properties", [key]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": sprintf("Resources.%s.Properties.StorageEncrypted should be defined", [key]),
+		"keyActualValue": sprintf("Resources.%s.Properties.StorageEncrypted is undefined", [key]),
+	}
+}
```

Comment:
The query "CMK Unencrypted Storage" already verifies that. I think it is not necessary to reverify the same checks in this query.

```suggestion
```